next/lib/get-params.ts

```diff
@@ -20,6 +20,7 @@ export function getNodePageJsonApiParams(resourceType: ResourceType) {
         "field_content_elements",
         "path",
         "status",
+        "metatag",
```

Comment:
this will need to be adapted in the zod branch (as the rest of changes in this file)

Reply:
Yep. I vote we merge this branch first and I can adapt mine after and handle the conflicts etc.